en/docs/tutorials/scenarios/scenario1-create-rest-api.md

```diff
@@ -1,4 +1,75 @@
+
+```
+
+Backend is working fine. Let’s expose this API through WSO2 API Manager. 
+
+1. Log in to https://localhost:9443/publisher/apis using Coltrain publisher user. Use user as _apiprovider@coltrain.com_ and password as _user123_
+2. Select **Import Open API** option under **REST API** section
+   
+    ![Create API]({{base_path}}/assets/img/tutorials/scenarios/create-api-oas-def.png)
+
+
+3. Select **OpenAPI File/Archive** radio button and import the **coltrain-public-openapi.yaml** definition from _resources_ location.
+
+    ![Import Definition]({{base_path}}/assets/img/tutorials/scenarios/import-oas-def.png)
+
+4. Add the context (say /coltrain) and create the API. Keep the endpoint as it is. You could click the tick in front of the endpoint to check the backend url status.
+
+    ![Create API]({{base_path}}/assets/img/tutorials/scenarios/create-coltrain-public-api.png)
+
+   
+Since this API is intended to be accessed by public users, Coltrain api providers want to remove any authentication for this resource. This way anyone can access the service without any credentials. For that,  
+
+1. Go to **Develop → API Configuration → Resources** tab
+2. Expand a resource
+3. Under the **Operation Governance** section, turn off the radio button for **Security**
+
+    ![Change security]({{base_path}}/assets/img/tutorials/scenarios/resource-security.png)
+
+4. Save the API.
+
+Before Publishing the API, Coltrain developers want to test this API. For that WSO2 API Manager provides a Test console to test the API while it is in the development stage.
+
+Before starting to test the API, you need to deploy the API in the gateway. For that, go to **Deploy → Deployment** section and select **Deploy** . This will deploy the API in the gateway but this won’t be visible to the outside world.
+
+![Deploy API]({{base_path}}/assets/img/tutorials/scenarios/deploy-coltrain-public.png)
+
+Now we are ready to test the API. For that, go to the **Test → Try Out** . This provides an API console in the Publisher ui to test the API.
+
+![Try out]({{base_path}}/assets/img/tutorials/scenarios/tryout.png)
+
+Expand the _/schedules_ resource and invoke the API. You won’t need to click **Generate Key** since we have removed the security for this resource. You will have to generate one if you want to test the _/schedules/{id}_ resource.
+
+![Try out result]({{base_path}}/assets/img/tutorials/scenarios/tryout-result.png)
+
+Now testing is done from the publisher end. The API needs to be published in-order to access it from the Developer portal. For that,
+
+1. Go to **Portal Configurations → Subscriptions** and select one from **Business Plans**. You could select “Unlimited” and save.
```

Comment:
```suggestion
1. Go to **Portal Configurations → Subscriptions** and select one from **Business Plans**. Select “Unlimited” and save.
```

Shall we expand on why we should select "Unlimited"?

Reply:
Updated the sentence. included unlimited as an example